blog/en/config/picked-posts.json

```diff
@@ -1,10 +1,10 @@
 [
     "blog/en/blog/2022/11/02/apache-apisix-v3-preview.md",
-    "blog/en/blog/2022/09/13/why-is-apache-apisix-the-best-api-gateway.md",
-    "blog/en/blog/2021/06/28/why-we-need-Apache-APISIX.md",
-    "blog/en/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md",
-    "blog/en/blog/2022/07/06/use-keycloak-with-api-gateway-to-secure-apis.md",
-    "blog/en/blog/2022/02/25/consul-api-gateway.md",
-    "blog/en/blog/2022/02/10/splunk-apisix-integration.md",
+    "blog/en/blog/2023/07/11/2023-apisix-meetup-malaysia.md",
+    "blog/en/blog/2023/06/02/lenovo-uses-apisix.md",
+    "blog/en/blog/2023/04/07/apisix-unity-group-q%26a.md",
```

Comment:
Should be `&` instead of `%26`, right?
```suggestion
    "blog/en/blog/2023/04/07/apisix-unity-group-q&a.md",
```
![image](https://github.com/apache/apisix-website/assets/36651058/6a3cd476-02ca-4c17-a588-8b7e441f4652)

![image](https://github.com/apache/apisix-website/assets/36651058/2a9b5d2d-409a-415d-8b8f-ec8e615d5007)